test/parallel/test-cluster-inspector-debug-brk.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+// Flags: --inspect={PORT}
+// This test ensures that flag --inspect-brk and --debug-brk works properly
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
+
+const debuggerPort = common.PORT;
+const script = common.fixturesDir + '/empty.js';
+
+if (cluster.isMaster) {
+
+  function fork(offset, execArgv) {
+    cluster.setupMaster(execArgv);
+    let worker = cluster.fork();
+
+    worker.on('exit', common.mustCall((code, signal) => {
+      assert.strictEqual(code, null);
+      assert.strictEqual(signal, 'SIGTERM');
+    }));
+
+    let socket = net.connect(debuggerPort + offset, common.mustCall(() => {
```

Comment:
I would wrap this code inside of a `worker.on('online', common.mustCall(...))` just to be sure that the worker is ready for the connection.